twoaxistracking/tests/test_layout.py

```diff
@@ -0,0 +1,161 @@
+from twoaxistracking import layout
+from shapely import geometry
+import numpy as np
+import pytest
+
+
+@pytest.fixture
+def square_field_layout_sloped():
```

Comment:
Fixtures can "inherit" from one another, so you could have this depend on `square_field_layout` if you wanted.  See https://docs.pytest.org/en/latest/how-to/fixtures.html#fixtures-can-request-other-fixtures

No preference from me which one is better here.  Just pointing out what's possible :)

Reply:
Neat. I have attempted to implement this, though I'm unsure if my method is the cleanest way.

Reply:
looks fine to me, that's how I'd do it 👍 

although the backslash/line continuation may not be necessary in this case?